script/build.js

```diff
@@ -195,16 +195,18 @@ if (options.watch) {
     // Check to see if we have a proxy config file
     const api = require('../config/config.proxy.js').api;
     devServerConfig.proxy = {
-      '/api/*': {
-        target: `https://${api.host}${api.path}`,
+     '/rx-api/*': {
+        target: `https://${api.host}/`,
         auth: api.auth,
-        rewrite: function (req) {
-          req.url = req.url.replace(/^\/api/, '');
+        secure: true,
+        changeOrigin: true,
+        pathRewrite: function(path, req){
```

Comment:
So, webpack docs are pretty terrible, and I can't figure out (1) whether this a webpack-specific function name or (2) you can name it whatever and webpack will match the signature to the proper rewriting function.

Also, nitpick: space before the curly.